haskell/cabal.bzl

```diff
@@ -109,6 +109,7 @@ def _prepare_cabal_inputs(hs, cc, dep_info, cc_info, component, package_id, tool
             "%{ghc_pkg}": hs.tools.ghc_pkg.path,
             "%{runghc}": hs.tools.runghc.path,
             "%{ar}": cc.tools.ar,
+            "%{cc}": cc.tools.cc,
```

Comment:
Why is this necessary in the context of an unix toolchain? Is this an "unrelated but fixed at the same time" issue?

Reply:
Yes, the comment that @regnat quoted is the reason for this change.